igb/repl.go

```diff
@@ -377,6 +507,40 @@ func newIVM() (ivm iVM, err error) {
 	return ivm, nil
 }
 
+// Returns true if double quotation in the string is open.
+func checkDoubleQuoteOpen(s string) bool {
```

Comment:
can't we extract duplicated logics from `checkDoubleQuoteOpen` and `checkSingleQuoteOpen`?

Reply:
Extracted the two lines

Reply:
I was hoping to see more extraction. The only differences of them are `^[^']*"`, `"\""` and `^[^"]*'`, `"'"`. I think we can make a generic function and pass these two differences as arguments to it.

Reply:
Thank you for pointing this out! Integrated into one function.